api/src/controllers/auth.js

```diff
@@ -159,10 +189,10 @@ exports.controllerFortyTwo = async (req, res) => {
                 );
             } else {
                 if (duplicateField === 'username') {
-                    res.status(400).redirect('/?error=username'); // Error for email already taken
+                    res.status(400).redirect('/?error=username'); // Error for username already taken
```

Comment:
These redirections lead the user to `http://localhost:8080/?error=...`, that is on the server root and not on the frontend.
This must be changed and the frontend must handle these cases.
It would be great if there was a frontend route taking the `error` param and showing some text to the user such as `The email you attempted to signing up with is already taken`.
I mention you @david-dacruz.

Reply:
The frontend part of this problem should not block the merging of this PR.